docs/source/Evaluation_Tutorial.md

```diff
@@ -46,7 +46,7 @@ You have to specify run parameters required by this pipeline:
 - `config`: Estimator config YAML file. You can use the default value which points to a YAML file packaged with our docker images or you can load from remote locations GCS or any HTTP(s) using file prefix `gs://, http(s)://`.
 - `tb_log_dir`: Path to store tensorboard logs used to visualize the training progress.
 - `checkpoint_dir`: Path to store output estimator checkpoints. These checkpoints represent your trained model and will be used for evaluation.
-- `volume_size`: Size of the Kubernetes Persistent Volume Claims (PVC) that will be used to store the dataset. You can use the default value.
+- `volume_size`: Size of the Kubernetes Persistent Volume Claims (PVC) that will be used to store the dataset. Normally, for 1K data, we need 3G volume.
```

Comment:
I will reword this to:

If you plan to use datasets from SynthDet(add url) project that include more than X images, you should increase this value accordingly. It requires 3GiB storage per 1K images.

Reply:
Update the tutorial.